src/js/utils/general-prop-types.js

```diff
@@ -15,6 +15,7 @@ export const colorPropType = PropTypes.oneOfType([
 export const backgroundPropType = PropTypes.oneOfType([
   PropTypes.string,
   PropTypes.shape({
+    clip: PropTypes.oneOfType([PropTypes.oneOf(['text']), PropTypes.string]),
```

Comment:
should we be specific in the values?
`border-box` `padding-box` `content-box`

Reply:
Good question. I was leaning into what we have done for other CSS types, where we have tended to try to cover the "80% case", and not have to explicitly enumerate every possible value. 

I do not have a strong opinion here. Curious to hear others thoughts.

Reply:
yeah Im also for either or just thinking outloud

Reply:
I'm good with following the pattern we've done elsewhere for now